www/tutorial/app/app.component.ts

```diff
@@ -68,7 +68,8 @@ export class AppComponent implements OnInit {
   }
 
   // called when tutorial commands need to be displayed
-  updateInstructions(step = null) {
```

Comment:
since we check the step value anyway, we do not have to set the default value, lets just use `updateInstructions(step)`